src/mono/mono/mini/mini-llvm.c

```diff
@@ -5636,8 +5636,11 @@ emit_llvmonly_handler_start (EmitContext *ctx, MonoBasicBlock *bb, LLVMBasicBloc
 		}
 	}
 
-	LLVMBuilderRef handler_builder = create_builder (ctx);
 	LLVMBasicBlockRef target_bb = ctx->bblocks [bb->block_num].call_handler_target_bb;
+	if (!target_bb)
+		return;
```

Comment:
can we assert something here?
it looks like we don't set `call_handler_target_bb` when the block is a filter clause, when it's an interp_entry cfg, or when we're already in a handler block?  looking at this loop:

https://github.com/dotnet/runtime/blob/42f3dceae23a5c7acb5d3cb8e0419e1195911956/src/mono/mono/mini/mini-llvm.c#L13405-L13426